src/Be.Vlaanderen.Basisregisters.Aws.DistributedMutex/DistributedLock.cs

```diff
@@ -104,13 +104,17 @@ public DistributedLock(DistributedLockOptions options, string lockName, ILogger
             _logger = logger;
             _lockName = lockName;
 
-            _mutex = CreateDynamoDbMutex(options);
+            _mutex = CreateMutex(options);
+            if (_mutex == null)
+            {
+                throw new NullReferenceException($"{nameof(CreateMutex)} result can't be null");
```

Comment:
throw ArgumentException instead of NullReferenceException

Reply:
done